### DISCUSSION TITLE
Phishing site hosted by Githib Pages

### DISCUSSION THREAD

**Discussion by sdpollack** [community/community#101505] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

This site is a phishing site.  https://pay-pal-customser-service.github.io/care/

There needs to be an more straightforward way to report fraudulent sites hosted on Github Pages.

**Comment by ettaboyle** [comment]
Thank you kindly for reaching out. If you are experiencing behavior against our Terms of Service or **need to report GitHub account-related abuse, you can reach out to our private support team.** We are unable to assist with or discuss personal account details here in Discussions.


Please visit this link for more details: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam We hope that helps!

**Comment by sdpollack** [comment]
None of the options on that page apply to this situation.  Someone is using Github Pages to host a phishing site.

I don't know who the user is, There is no issue or pull request to report from.  This is not from a discussion or comment.  I looked for the app in Marketplace but it was not listed.  It is not a contact link.

I see no way to report abuse in this situation.  Clicking the various Contact Support links do not lead to a way to report the issue, they only lead back to knowledge bases or the community site.  Please advise.
